linkerd/app/outbound/src/lib.rs

```diff
@@ -466,20 +468,25 @@ impl Config {
             .push(metrics.clone().http_handle_time.layer());
 
         let http_server = svc::stack(http_router)
-                .check_make_service::<Logical<HttpEndpoint>, http::Request<_>>()
-                .push_make_ready()
-                .push_timeout(dispatch_timeout)
-                .push(router::Layer::new(LogicalPerRequest::from))
-                // Used by tap.
-                .push_http_insert_target()
-                .push_on_response(http_admit_request)
-                .push_on_response(metrics.stack.layer(stack_labels("source")))
-                .instrument(
-                    |src: &tls::accept::Meta| {
-                        info_span!("source", target.addr = %src.addrs.target_addr())
-                    },
-                )
-                .check_new_service::<tls::accept::Meta>();
+            // Resolve the application-emitted destination via DNS to determine
+            // it's canonical FQDN to use for routing.
```

Comment:
nit: "it's" == "it is"; i think this should be
```suggestion
            // its canonical FQDN to use for routing.
```